### PR TITLE
Add Azul Zulu 14+36 (v14.27.1)

### LINKED ISSUE #69: Introduce Azul Zulu JDK 14.x Releases

### PR DESCRIPTION
**Pull request by x80486**
Closes #69

### CODE REVIEW

**Comment by x80486** [comment]
**Suggestion:** I think it will be better if you use `fast-forward merges` (I think it's the third option offered), so no merge commits are created and the history remains cleaner and clear :wink: :sunglasses:

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#
```
Comment (joschi) [inline .gitignore]: These entries should probably live in your global `gitignore` file.

https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Comment (halcyon) [reply .gitignore:1]: @x80486 I think the suggestion from @joschi is a good one. If you would be so kind as to remove the .gitignore, I'll merge your PR ASAP. Thank you!

File: zulu/zulu.json [hunk 2]
```diff
@@ -1,4 +1,28 @@
...
+    "release_name": "azul-zulu-14.27.1-jdk14",
+    "binaries": [
+      {
+        "os": "linux",
+        "architecture": "x64",
+        "heap_size": "normal",
+        "jvm_impl": "hotspot",
+        "package": {
+          "link": "https://cdn.azul.com/zulu/bin/zulu14.27.1-ca-jdk14-linux_x64.tar.gz",
+          "checksum_link": "https://raw.githubusercontent.com/halcyon/asdf-java/master/zulu/zulu14.27.1-ca-jdk14-linux_x64.tar.gz.sha256.txt"
```
Comment (halcyon) [inline zulu/zulu.json:12]: I just merged a PR that prefers that these be `checksum` and that the checksum be the value

**Comment by halcyon** [comment]
Merged via https://github.com/halcyon/asdf-java/commit/0f60aebb10beaf4e7cc597de82ebf553be61d6d8
Thank you!

**Pull request closed without merging**